.gitlab-ci.yml

```diff
@@ -113,8 +127,14 @@ MC-hls-build:
     VIVADO_VERSION: "2019.2"
     PROJ_NAME: "MC"
 # FW sim_synth ---------------
-topPRMEMC-sim_synth:
-  <<: *template_topTF-sim_synth
+topPRMEMC-sim:
+  <<: *template_topTF-sim
+  variables:
+    VIVADO_VERSION: "2019.2"
+    PROJ_NAME: "PRMEMC"
+# FW sim_synth ---------------
```

Comment:
Adjust comment

Reply:
Done. Changed to "# FW synthesis ------------"